Example/Controllers/ViewController.swift

```diff
@@ -122,8 +125,34 @@ extension ViewController: UITableViewDelegate, UITableViewDataSource {
 // MARK: - SettingsDelegate
 extension ViewController: SettingsDelegate {
     func didSettingsUpdated(controller: SettingsTableViewController, updated miniAppList: [MiniAppInfo]?) {
-        self.decodeResponse?.removeAll()
-        self.decodeResponse = miniAppList
+        self.unfilteredResults?.removeAll()
+        self.unfilteredResults = miniAppList
+        self.tableView.reloadData()
+    }
+}
+
+extension ViewController: UISearchBarDelegate {
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
```

Comment:
I am not sure, if only `searchButtonClicked` naming is good. Feel free about my suggestion :D

Reply:
this is Apple UISearchBarDelegate protocol method, we should send them a ticket :)

Reply:
Sorry, I [got it.](https://developer.apple.com/documentation/uikit/uisearchbardelegate/1624294-searchbarsearchbuttonclicked?language=objc) Please ignore my comment.